packages/perseus/src/types.ts

```diff
@@ -24,11 +25,66 @@ export type Dimensions = {
 
 export type DeviceType = "phone" | "tablet" | "desktop";
 
-// TODO(CP-4839): Create a proper type for Widget
-// Is this the same as the Widget type in `renderer.jsx`?
-export type Widget = any;
+export type SerializedState = {
+    [id: string]: any;
+};
```

Comment:
This is another non-obvious part of Perseus. The `Renderer` will return the widget's render props by default. But, each widget can provide its own `getSerializedState` and `restoreSerializedState` functions which return/consume anything. 

I think the best way to solve this would be to make `Widget` a generic that defined a type for what this pair of functions returns/consumes, but I'd like to avoid that for now. 

I think what I'll do for the short term is to change this type to be a `Record<string, any>` as that feels like the more generic/obvious way to say "it's an object".